portfolio/src/main/webapp/script.js

```diff
@@ -39,6 +39,21 @@ function addRandomGreeting() {
  */
 async function fetchContent() {
   const response = await fetch('/data');
-  const commentText = await response.text();
-  document.getElementById('content-container').innerHTML = commentText;
+  const messages = await response.json();
+  const messagesElement = document.getElementById('content-container');
+  messagesElement.innerHTML = '';
+  for (message of messages) {
+    messagesElement.appendChild(createListElement(message));
+  }
+}
+
+/**
+ * Creates an <li> element containing text.
+ * @param text the text to create list element with
+ * @return a list element
```

Comment:
This should look like
`@param {type} paramName description`

see example here: https://google.github.io/styleguide/jsguide.html#jsdoc-line-wrapping